src/components/navigation-header/index.tsx

```diff
@@ -64,13 +61,11 @@ const AuthenticationControls = () => {
 						? getUserMenuItems({ signOut })
 						: [
 								{
-									icon: <IconSignIn16 />,
```

Comment:
💭 makes sense re: assembly given the removal of icons... if we wanted to though, could we use `__config.flags.enable_dark_mode` to conditionally show these icons?

Something like:

```jsx
icon: __config.flags.enable_dark_mode ? undefined : <IconSignIn16 />
```

Note sure if that would work, just tossing it out there in case it might 😬

Reply:
Good point, In a future PR though I'll need to refactor the placement of the icons so the assembly felt like an easier thing to handle given that we want to ship in a week or so.